Sakshi/hashtable.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+struct pair{
+    int value;
+    int key;
+};
+void display(struct pair[], int size)
+int 
+main()
+{
+    int size,i,temp;
+    printf("enter the size of the table");
+    scanf("%d",&size);
+    
+    struct pair hash_table[size];
+    
+    printf("enter the element");
+    for(i=0; i<size; i++)
+    {
+        scanf("%d",&temp);
```

Comment:
dude, i think instead of storing directly using mode function you should make another function for storing nodes into hash-table , as hashing-function may not always be "key%size"  and all you did here is for every number that you take you are pairing it with the node and not forming any contiguous links ,that means if i take size = 10 , and elements to be 32, 42,52 your hashtable will only store 52 rest every inputs will be lost , this is happening as you have taken hash to be an array of structure pair and not array of linked list